README.md

```diff
@@ -89,6 +89,9 @@ The commands to build the Locale are started from the main directory of the proj
 make clean
 make -e SPHINXOPTS="-D language='de'" BUILDDIR="build/de" html
 ```
+"""
```

Comment:
Please change these to back ticks: same as in the code section above.

Reply:
changed to back ticks